examples/mnist/Instructions.md

```diff
@@ -0,0 +1,27 @@
+<p align="center"><b>Setup, build and run tiny-dnn on various operating systems</b><p align="center">
+
+<b>tiny-dnn</b> is a C++11 implementation of deep learning. It is suitable for deep learning on limited computational resource, embedded systems and IoT devices.
+
+
```

Comment:
@karandeepdps Please remove numbered lists markdown attached to shell commands.

You can check current `Instructions.md` here.
https://github.com/karandeepdps/tiny-dnn/blob/ee4c4a3b6324e55690924d2056f3b3bcd1d50dc2/examples/mnist/Instructions.md

If you would remove the numbers before shell commands, people can simply copy shell commands and paste them to terminal. So it's more convenient to use.